pages/create-release.js

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from 'react'
+import { supabase } from '@/utils/supabase'
+import { useUser } from '@/utils/context/user'
+
+const trueFalse = [
+  { id: 1, text: 'True', value: true },
+  { id: 2, text: 'False', value: false }
+]
+
+const releaseTypes = [
+  { id: 1, text: 'LP' },
+  { id: 2, text: 'EP' },
+  { id: 3, text: 'Single' },
+  { id: 4, text: 'Compilation' },
+  { id: 5, text: 'Soundtrack' },
+  { id: 6, text: 'Choose release type', isDisabled: true }
+]
+
+export default function CreateRelease() {
+  const { activeUser } = useUser()
+  const [newRelease, setNewRelease] = useState({
+    title: '',
+    artist: '',
+    label: '',
+    artwork: '',
+    downloadUrl: '',
+    pagePassword: '',
+    isPasswordProtected: trueFalse[1].value,
+    type: releaseTypes[5].text,
+    userId: ''
+  })
+
+  useEffect(() => {
+    if (activeUser) {
+      if (activeUser.type === 'Label') {
+        setNewRelease({
+          ...newRelease,
+          label: activeUser.name,
+          userId: activeUser.id
+        })
+      }
+      if (activeUser.type === 'Artist') {
+        setNewRelease({
+          ...newRelease,
+          artist: activeUser.name,
+          userId: activeUser.id
+        })
+      }
```

Comment:
It shouldn't matter if it's a label or an artist (or a moonwalker), they should share the same logic

Reply:
Would you recommend using something like this?

`setNewRelease({
   ...newRelease,
   label: activeUser.type === 'Label' ? activeUser.name : null,
   artist: activeUser.type === 'Artist' ? activeUser.name : null,
   userId: activeUser.id
  })
`